themes/custom/droopler_theme/scss/components/_d_p_counters.scss

```diff
@@ -17,6 +17,10 @@ $d-p-counters-padding-top: 7.5rem !default;
 $d-p-counters-padding-bottom: 7.5rem !default;
 $d-p-counters-mobile-padding-top: 3rem !default;
 $d-p-counters-mobile-padding-bottom: 3rem !default;
+$d-p-single-counter-padding-left: 5% !default;
```

Comment:
Tak z ciekawości - czemu potrzebujemy paddingu w poziomie? Nie załatwia go Bootstrap?

Reply:
Jak masz tło czy obraz to wtedy musi być lekki padding bo jest tak szerokie jak kolumna. Rzeczy Bootstrapowe są wcześniej. Pewnie jakoś by się dało ale bym musiał ostro hackować, znów przenosić tło do jakiegoś aftera i pozycjonować. Jak coś mogę kombinować dziś ale jak się nie uda to bym na ten czas zostawił jak jest bo wygląda to ok.

Reply:
jak to są takie same paddingi, to może warto zrobić jedną zmienną `$d-p-single-counter-padding-x` ?

Reply:
Spoko, mogę tak zrobić.